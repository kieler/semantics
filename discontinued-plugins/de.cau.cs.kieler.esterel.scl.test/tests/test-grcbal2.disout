module test_grcbal2:
input A;
input B;
output C;
output D;
output E;

loop
  if (A) {
    [
      emit C
    ||
      if (D) {
        
      } else {
        emit E
      }
    ]
  } else {
    if (B or C) {
      emit D
    }
  };
  pause
end loop

end module
