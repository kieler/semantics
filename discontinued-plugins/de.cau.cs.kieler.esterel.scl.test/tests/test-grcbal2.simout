   0 C=0 D=0 E=0 
   1 C=1 D=0 E=1 
   2 C=1 D=0 E=1 
   3 C=1 D=0 E=1 
   4 C=0 D=0 E=0 
   5 C=0 D=0 E=0 
   6 C=1 D=0 E=1 
   7 C=1 D=0 E=1 
   8 C=1 D=0 E=1 
   9 C=0 D=0 E=0 
  10 C=1 D=0 E=1 
  11 C=1 D=0 E=1 
  12 C=0 D=1 E=0 
  13 C=0 D=1 E=0 
  14 C=0 D=0 E=0 
  15 C=0 D=1 E=0 
  16 C=1 D=0 E=1 
  17 C=1 D=0 E=1 
  18 C=1 D=0 E=1 
  19 C=1 D=0 E=1 
  20 C=0 D=0 E=0 
  21 C=1 D=0 E=1 
  22 C=1 D=0 E=1 
  23 C=1 D=0 E=1 
  24 C=0 D=0 E=0 
  25 C=1 D=0 E=1 
  26 C=0 D=1 E=0 
  27 C=0 D=0 E=0 
  28 C=1 D=0 E=1 
  29 C=0 D=0 E=0 
  30 C=0 D=0 E=0 
  31 C=0 D=0 E=0 
  32 C=1 D=0 E=1 
  33 C=1 D=0 E=1 
  34 C=0 D=0 E=0 
  35 C=0 D=1 E=0 
  36 C=0 D=0 E=0 
  37 C=1 D=0 E=1 
  38 C=0 D=0 E=0 
  39 C=0 D=1 E=0 
  40 C=0 D=1 E=0 
  41 C=1 D=0 E=1 
  42 C=1 D=0 E=1 
  43 C=1 D=0 E=1 
  44 C=1 D=0 E=1 
  45 C=0 D=0 E=0 
  46 C=1 D=0 E=1 
  47 C=1 D=0 E=1 
  48 C=0 D=0 E=0 
  49 C=0 D=0 E=0 
  50 C=0 D=0 E=0 
  51 C=0 D=0 E=0 
  52 C=1 D=0 E=1 
  53 C=1 D=0 E=1 
  54 C=1 D=0 E=1 
  55 C=1 D=0 E=1 
  56 C=1 D=0 E=1 
  57 C=0 D=0 E=0 
  58 C=1 D=0 E=1 
  59 C=0 D=1 E=0 
  60 C=0 D=0 E=0 
  61 C=1 D=0 E=1 
  62 C=1 D=0 E=1 
  63 C=1 D=0 E=1 
  64 C=1 D=0 E=1 
  65 C=1 D=0 E=1 
  66 C=0 D=1 E=0 
  67 C=1 D=0 E=1 
  68 C=0 D=0 E=0 
  69 C=0 D=0 E=0 
  70 C=0 D=1 E=0 
  71 C=0 D=1 E=0 
  72 C=1 D=0 E=1 
  73 C=1 D=0 E=1 
  74 C=1 D=0 E=1 
  75 C=0 D=0 E=0 
  76 C=1 D=0 E=1 
  77 C=0 D=0 E=0 
  78 C=0 D=0 E=0 
  79 C=1 D=0 E=1 
  80 C=1 D=0 E=1 
  81 C=1 D=0 E=1 
  82 C=1 D=0 E=1 
  83 C=0 D=1 E=0 
  84 C=0 D=0 E=0 
  85 C=1 D=0 E=1 
  86 C=1 D=0 E=1 
  87 C=1 D=0 E=1 
  88 C=0 D=1 E=0 
  89 C=1 D=0 E=1 
  90 C=1 D=0 E=1 
  91 C=1 D=0 E=1 
  92 C=1 D=0 E=1 
  93 C=1 D=0 E=1 
  94 C=1 D=0 E=1 
  95 C=0 D=0 E=0 
  96 C=1 D=0 E=1 
  97 C=1 D=0 E=1 
  98 C=0 D=1 E=0 
  99 C=1 D=0 E=1 
