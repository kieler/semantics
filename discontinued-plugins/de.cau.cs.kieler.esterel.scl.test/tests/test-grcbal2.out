module test_grcbal2:
input A;
input B;
output C;
output D;
output E;

loop
  present A then
    [
      emit C
    ||
      present D then
        nothing
      else
        emit E
      end present
    ]
  else
    present [B or C] then
      emit D
    end present
  end present;
  pause
end loop

end module
